src/main/scala/org/allenai/scholar/StructuredDocument.scala

```diff
@@ -0,0 +1,21 @@
+package org.allenai.scholar
+
+import org.allenai.scholar.metrics.metadata.PaperMetadata
+
+/** Created by rodneykinney on 4/29/15.
+  */
+case class StructuredDocument(
```

Comment:
This is the replacement for MetadataAndBibliography.  The main addition is that each bib-item is associated with a list of mentions in the text.